packages/orm/src/QueryParser.ts

```diff
@@ -45,11 +300,14 @@ export function parseEntityFilter(meta: EntityMetadata<any>, filter: any): Parse
  * The ValueFilter is a
  */
 export type ParsedValueFilter<V> =
-  | { kind: "eq"; value: V | null }
```

Comment:
It ended up being cleaner to break `null` out as its own `is-null` / `not-null` kinds.